src/lib/core/option/option.ts

```diff
@@ -161,11 +161,11 @@ export class MatOption implements AfterViewChecked, OnDestroy {
   }
 
   /** Sets focus onto this option. */
-  focus(): void {
+  focus(options?: FocusOptions): void {
```

Comment:
There's something in the DOM API that's called the same. I think we added ours since the interface isn't there in earlier TS versions (which probably happened in G3). I've added the import.